### PR TITLE
Automate installation and compilation of Lotus

### PR DESCRIPTION
**Pull request by matejpavlovic**

### CODE REVIEW

**Comment by matejpavlovic** [comment]
@adlrocha you probably don't need to go through all the playbooks and scripts, I'm mostly curious whether the README makes sense to you. I'll merge after your approval.

File: deployment/README.md
Comment (adlrocha) [inline deployment/README.md:3]:
```suggestion
We use [Ansible](https://www.ansible.com/) to deploy Spacenet nodes (and whole networks).
```

File: deployment/README.md
Comment (adlrocha) [inline deployment/README.md:26]:
```suggestion
A reference of the provided deployment playbooks is given at the end of this document.
```

File: deployment/README.md
Comment (adlrocha) [inline deployment/README.md:6]: As discussed sync, we may need to support for >1 bootstrap.

File: deployment/README.md
Comment (adlrocha) [inline deployment/README.md:43]:
```suggestion
- Sudo access without password on remote machines.
```
Can this be relaxed?

File: deployment/README.md
Comment (adlrocha) [inline deployment/README.md:170]: Is `generate-genesis.yaml` triggered every time we call `start-bootstrap.yaml`, or if there is an existing genesis already it just distributed and apply that one? I ask because re-generating every time the bootstrap may lead to non-determinism and errors as the ones we've seen when restarting single nodes of the network.

**Review by adlrocha — APPROVED** [review]
I haven't looked at the playbooks in depth. I read the README and reviewed exclusively the things that could lead to potential bugs. I can't accept suggestions, feel free to address the suggestions that you think make sense and merge this PR. We can defer addressing some of the comments in the next PR once we have support for fixed address bootstraps. Thanks :pray:

File: deployment/README.md
Comment (matejpavlovic) [reply deployment/README.md:170]: The genesis is not generated every time.

**Pull request merged**
